Makefile

```
@@ -544,6 +544,11 @@ ray-project-mini-image-build:
 		$(IMAGE_BUILD_EXTRA_OPTS) \
 		-f ./hack/testing/ray-mini/Dockerfile ./ \
 
+.PHONY: ray-project-mini-image-build-push
+kind-ray-project-mini-image-build: PLATFORMS=$(HOST_IMAGE_PLATFORM)
+ray-project-mini-image-build-push: PUSH=--push
+ray-project-mini-image-build-push: ray-project-mini-image-build
```

checkmake (0.3.2)

[warning] Target "ray-project-mini-image-build-push" defined multiple times (lines 549 and 550).

(uniquetargets)